platform/logging/layouts/JsonLayout.ts

```diff
@@ -0,0 +1,30 @@
+import { Schema, typeOfSchema } from '../../types';
+import { LogRecord } from '../LogRecord';
+import { Layout } from './Layouts';
+
+const createSchema = ({ literal, object }: Schema) => {
+  return object({
+    kind: literal('json')
+  });
+};
+
+const schemaType = typeOfSchema(createSchema);
+/** @internal */
+export type JsonLayoutConfigType = typeof schemaType;
+
+/**
+ * Layout that just converts `LogRecord` into JSON string.
+ * @internal
+ */
+export class JsonLayout implements Layout {
+  static createConfigSchema = createSchema;
+
+  format(record: LogRecord): string {
+    // Per spec `timestamp.toJSON()` uses `timestamp.toISOString()` under the hood.
+    // See http://www.ecma-international.org/ecma-262/5.1/#sec-15.9.5.44
+    return JSON.stringify({
```

Comment:
Yeah, will make it closer to what we have now in Kibana (it will be trivial to tweak it later anyway):
```json
{  
   "type":"log",
   "@timestamp":"2017-07-31T12:08:04Z",
   "tags":[  
      "status",
      "plugin:timelion@6.0.0-beta1",
      "info"
   ],
   "pid":32047,
   "state":"green",
   "message":"Status changed from uninitialized to green - Ready",
   "prevState":"uninitialized",
   "prevMsg":"uninitialized"
}
```
and here is what we have in Elasticsearch:
```json
{
  "timeMillis" : 1501510600593,
  "thread" : "Thread-2",
  "level" : "INFO",
  "loggerName" : "org.elasticsearch.node.Node",
  "marker" : {
    "name" : "[SRbYZv9] "
  },
  "message" : "closed",
  "endOfBatch" : false,
  "loggerFqcn" : "org.apache.logging.log4j.spi.AbstractLogger",
  "threadId" : 15,
  "threadPriority" : 5
}
```